package.json

```diff
@@ -236,6 +236,7 @@
     "validate-ts": "gulp validate-ts",
     "internal-tool": "dx-tools",
     "validate-declarations": "dx-tools validate-declarations --js-scripts ./js --exclude js/renovation/",
+    "testcafe-in-docker": "docker build -f ./testing/testcafe/docker/Dockerfile -t testcafe-testing . && docker run -it testcafe-testing",
```

Comment:
For why param `-it` here?

Reply:
`-t` for attaching the bash process to terminal
`-i` for being able to send inputs via STDIN to the bash in the container
For example: stop executing tests or pass `--componentFolder` param in process